src/vs/workbench/contrib/files/test/browser/explorerModel.test.ts

```diff
@@ -12,37 +12,39 @@ import { ExplorerItem } from 'vs/workbench/contrib/files/common/explorerModel';
 import { toResource } from 'vs/base/test/common/utils';
 import { TestFileService, TestPathService } from 'vs/workbench/test/browser/workbenchTestServices';
 import { TestConfigurationService } from 'vs/platform/configuration/test/common/testConfigurationService';
+import { IConfigurationService } from 'vs/platform/configuration/common/configuration';
+import { SortOrder } from 'vs/workbench/contrib/files/common/files';
 
 
 suite('Files - View Model', function () {
 
 	const fileService = new TestFileService();
-	const configService = new TestConfigurationService();
-	function createStat(this: any, path: string, name: string, isFolder: boolean, hasChildren: boolean, size: number, mtime: number): ExplorerItem {
+	const testConfigService = new TestConfigurationService();
+	function createStat(this: any, path: string, name: string, isFolder: boolean, hasChildren: boolean, size: number, mtime: number, configService: IConfigurationService): ExplorerItem {
 		return new ExplorerItem(toResource.call(this, path), fileService, configService, undefined, isFolder, false, false, name, mtime);
 	}
 
 	const pathService = new TestPathService();
 
 	test('Properties', function () {
 		const d = new Date().getTime();
-		let s = createStat.call(this, '/path/to/stat', 'sName', true, true, 8096, d);
```

Comment:
could we not make all these changes? Seems like a default parameter value should do what you want with a lot less churn

Reply:
For sure. I think the same could be said for the other Params too, but if the goal is avoiding line changes I won't change those.

Reply:
Removed.